internal/operands/template-validator/resources.go

```diff
@@ -37,6 +35,7 @@ const (
 	ServiceName                   = VirtTemplateValidator
 	MetricsServiceName            = "template-validator-metrics"
 	DeploymentName                = VirtTemplateValidator
+	ConfigMapName                 = VirtTemplateValidator
```

Comment:
Use `VirtTemplateValidator` directly?

Reply:
I wanted to keep the pattern as other constants. I can change it if you want.

Reply:
Let's stay consistent for now. It could be changed in a follow up.